specialization_week_1/specialization_week_1.py

```diff
@@ -0,0 +1,115 @@
+"""
```

Comment:
✅ Great job on Task 1
Point 7.5/8

-0.5 for unnecessary member attributes i.e. `self.item` and `self.price`